src/FEValues/common_values.jl

```diff
@@ -103,9 +113,12 @@ where ``u_i`` are the value of ``u`` in the nodes. For a vector valued function
 ``\\mathbf{u}(\\mathbf{x}) = \\sum\\limits_{i = 1}^n N_i (\\mathbf{x}) \\mathbf{u}_i`` where ``\\mathbf{u}_i`` are the
 nodal values of ``\\mathbf{u}``.
 """
-function function_value(fe_v::Values{dim}, q_point::Int, u::AbstractVector{T}, dof_range = eachindex(u)) where {dim,T}
+function_value(fe_v::T, q_point, u, dof_range) where T = function_value(FieldTrait(T), fe_v, q_point, u, dof_range)
+function_value(fe_v::T, q_point, u) where T = function_value(FieldTrait(T), fe_v, q_point, u)
+
+function function_value(ft::FieldTrait, fe_v::Values{dim}, q_point::Int, u::AbstractVector{T}, dof_range = eachindex(u)) where {dim,T}
     n_base_funcs = getn_scalarbasefunctions(fe_v)
-    isa(fe_v, VectorValues) && (n_base_funcs *= dim)
+    isa(ft, IsVectorValued) && (n_base_funcs *= dim)
```

Comment:
Sorry for the question, but isn't this just `getnbasefunctions` ?

Reply:
Good point, I think `getnbasefunctions` should do the job here. Any reason that wasn't used @fredrikekre ?

Reply:
Maybe, if the result of that function has changed since https://github.com/Ferrite-FEM/Ferrite.jl/commit/894f4d523a4a3371868ae1f66df57d9746bd1b7a

Reply:
It has in #125